libcxx/test/benchmarks/ContainerBenchmarks.h

```diff
@@ -119,6 +119,90 @@ void BM_InsertValueRehash(benchmark::State& st, Container c, GenInputs gen) {
   }
 }
 
+// Wrap any Iterator into an input iterator
+template <typename Iterator>
+class InputIterator {
```

Comment:
I think you could reuse the type from `test_iterators.h` like you've done in some of your recent patches.

Reply:
Yeah, I totally agree. This is one of my earliest PRs. During that time, I was not aware of the test utilities we already have. Now I have better understandings on the test framework libc++ offered, and will reuse the code as much as possible. In the meantime, I will try to improve this early work and provide an updated benchmark result.